lib/iris/tests/results/netcdf/netcdf_save_gridmapmulti.cdl

```diff
@@ -15,12 +15,12 @@ variables:
 		latitude_longitude:earth_radius = 6371229. ;
 	int64 longitude(longitude) ;
 		longitude:axis = "X" ;
-		longitude:units = "degrees_east" ;
+		longitude:units = "1" ;
```

Comment:
It appears the units of these coords are '1', so they fail the if check. I'll look into why they have units of '1'.